boards/st/nucleo_h745zi_q/Kconfig.defconfig

```diff
@@ -0,0 +1,15 @@
+# STM32H745ZI Nucleo board configuration
+
+# Copyright (c) 2020 Alexander Kozhinov <AlexanderKozhinov@yandex.com>
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_NUCLEO_H745ZI_Q_STM32H745XX_M7 || BOARD_NUCLEO_H745ZI_Q_STM32H745XX_M4
```

Comment:
For these you can just do `if BOARD_NUCLEO_H745ZI_Q`

Reply:
indeed, txs